File: PYTHON/SivitasAkademik.py
class SivitasAkademik:
    # contructor, Inisialisasi atribut
    def __init__(self, asal_universitas, email_edu):
        self._asal_universitas = asal_universitas
        self._email_edu = email_edu

    # setter dan getter
    def getAsalUniversitas(self):
        return self._asal_universitas

    def setAsalUniversitas(self, asal_universitas):
        self._asal_universitas = asal_universitas

    def getEmailEdu(self):
        return self._email_edu

    def setEmailEdu(self, email_edu):
        self._email_edu = email_edu

File: C++/SivitasAkademik.cpp
#include <iostream>
#include <string>
using namespace std;
#include "SivitasAkademik.h"

// constructor,Inisialisasi atribut
SivitasAkademik::SivitasAkademik(const string& asal_universitas, const string& email_edu) {
    this->_asal_universitas = asal_universitas;
    this->_email_edu = email_edu;
}


// getter dan setter
string SivitasAkademik::getAsalUniversitas() {
    return _asal_universitas;
}

void SivitasAkademik::setAsalUniversitas(string asal_universitas) {
    _asal_universitas = asal_universitas;
}

string SivitasAkademik::getEmailEdu() {
    return _email_edu;
}

void SivitasAkademik::setEmailEdu(string email_edu) {
    _email_edu = email_edu;
}
